src/components/TitleAndMetaTags/TitleAndMetaTags.js

```diff
@@ -11,7 +11,15 @@ import React from 'react';
 
 const defaultDescription = 'A JavaScript library for building user interfaces';
 
-const TitleAndMetaTags = ({title, ogDescription, ogUrl}) => {
+const TitleAndMetaTags = ({
+  title,
+  ogDescription,
+  ogUrl,
+}: {
+  title: string,
+  ogDescription: string,
+  ogUrl: string,
+}) => {
```

Comment:
This may be a subjective nit, but prefer types to be declared separately (for readability), eg:
```js
type Props = {
  title: string,
  ogDescription: string,
  ogUrl: string,
};

const TitleAndMetaTags = ({ title, ogDescription, ogUrl }: Props) => {
```

Reply:
No problem with me on that :)